test/e2e/fleet_test.go

```diff
@@ -56,7 +56,10 @@ func TestFleetScaleUpEditAndScaleDown(t *testing.T) {
 	fixtures := []bool{true, false}
 
 	for _, usePatch := range fixtures {
+		usePatch := usePatch
```

Comment:
I figure someone will ask about this:
https://gist.github.com/posener/92a55c4cd441fc5e5e85f27bca008721